dbld/images/centos7/Dockerfile

```diff
@@ -3,11 +3,13 @@ MAINTAINER Andras Mitzki <andras.mitzki@balabit.com>
 
 RUN yum install -y wget epel-release
 
-RUN cd /etc/yum.repos.d && wget https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng39/repo/epel-7/czanik-syslog-ng39-epel-7.repo
+RUN cd /etc/yum.repos.d && wget https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng311/repo/epel-7/czanik-syslog-ng311-epel-7.repo
 
 ADD dev-dependencies.txt .
 RUN cat dev-dependencies.txt | grep -v "#" | xargs yum -y  install
-
+ADD gradle_installer.sh .
+RUN ./gradle_installer.sh
+RUN find / -name libjvm.so | xargs dirname | tee --append /etc/ld.so.conf.d/openjdk-libjvm.conf && ldconfig
```

Comment:
Yes, the file generated on different locations depends on java-1.8.0-openjdk-devel version. The java version is in directory name. Example: 
`/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.141-1.b16.el7_3.x86_64/jre/lib/amd64/server`